lib/compilers/rust.js

```diff
@@ -24,6 +24,7 @@
 
 import path from 'path';
 
+import fs from 'fs-extra';
```

Comment:
Unused import according to CI
```suggestion
```

Reply:
right, not neede after moving the methods back in the base class...